doc/moveit_cpp/src/moveit_cpp_tutorial.cpp

```diff
@@ -78,7 +77,7 @@ int main(int argc, char** argv)
 
   Eigen::Isometry3d text_pose = Eigen::Isometry3d::Identity();
   text_pose.translation().z() = 1.75;
-  visual_tools.publishText(text_pose, "MoveItCpp Demo", rvt::WHITE, rvt::XLARGE);
+  visual_tools.publishText(text_pose, "MoveItCpp_Demo", rvt::WHITE, rvt::XLARGE);
```

Comment:
Any reason we are changing it to not include space but instead `_`?

Reply:
This is question also applicable for other related changes as well.

Reply:
I added the `_` to all of the displayed text because visual_tools puts a ridiculous amount of space between the words
![image](https://user-images.githubusercontent.com/25058794/124193751-a09b4280-da84-11eb-964e-901cbd8f9904.png)